docs/typescript.md

```diff
@@ -881,9 +881,16 @@ export class ComposableController<
   const controllerMessenger = ControllerMessenger<any, any>;
   ```
 
+### Type-Only Dependencies
+
+If package `a` imports only types from `b`, `b` be a dev or production dependency of `a`?
```

Comment:
```suggestion
If package `a` imports only types from package `b`,  should `b` be a dev or production dependency of `a`?
```

Reply:
Going with @mcmire under the logic that it's clear from the context that `b` is a package.